### DISCUSSION TITLE
How to stop a bunch?

### DISCUSSION THREAD

**Discussion by hepabolu** [ttscoff/bunch-docs#283] [Q&A]
Following https://github.com/ttscoff/bunch/discussions/278 I'm trying to tell Bunch to stop executing. What I want to achieve is:

```
---
title: Project specific setups
---

project=?{
    Project A => P1
    Project B => P2
    Project C => P2
} "Which project"

# if no project is selected
if not project
	<<#Close
end

<<#${project}

(notify this should not be visible when cancelled) 

___
---[P1]

(notify project ${project} is selected)
* say "starting setup for P1"

---[P2]

(notify project ${project} is selected)
* say "starting context for P2"

---[Close]

(notify dialog cancelled)
* say "Good bye"
```

What happens is that after the [Close] fragment the last notify is also executed. How can I stop the bunch in the [Close] snippet?

**Comment by ttscoff** [comment]
You can’t halt the execution, but you can put the last two statements in an else block after the ‘if not project’ logic.

- Brett
